src/components/booleanDropdown.tsx

```diff
@@ -0,0 +1,14 @@
+import Dropdown from 'react-dropdown';
+import '../styles/booleanDropdown.scss';
+
+const options = ['True', 'False'];
+
+interface Props {
+  change: () => void;
+}
+
+function BooleanDropdown(props:Props):JSX.Element  {
+  return <Dropdown className='drop' options={options} value='False' arrowClassName='arrow' onChange={() => props.change()}/>;
```

Comment:
you can just do this!
```suggestion
  return <Dropdown className='drop' options={options} value='False' arrowClassName='arrow' onChange={props.change}/>;
```